src/Compiler/Symbols/Symbols.fs

```diff
@@ -2245,6 +2258,11 @@ type FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
         | V vref -> isForallFunctionTy cenv.g vref.Type
         | _ -> false
 
+    member x.IsRefCell =
```

Comment:
I think I prefer the name `IsFSharpRefType` for the public API

Reply:
It's possible, yes, but it'd be less convenient. We've been using this API for several years, and I found it useful. Will you accept it with  `IsFSharpRef` name?

Reply:
Yes, I guess it's OK given the special role the type plays in some corner-case F# language rules